src/main/java/com/redhat/rcm/offliner/Main.java

```diff
@@ -346,7 +372,8 @@ private ArtifactListReader getArtifactListReader( File file )
 
                                     if ( !original.equals( current ) )
                                     {
```

Comment:
Again, we should be careful about missing checksums in the artifact list, right?